class CalculationError(RuntimeError):
    """Exceção para erros durante os cálculos."""
    pass